Makefile

```diff
@@ -25,10 +29,10 @@ INCLUDES	:=
 # options for code generation
 #---------------------------------------------------------------------------------
 
-CFLAGS	= -g -O2 -Wall $(MACHDEP) $(INCLUDE)
-CXXFLAGS	=	$(CFLAGS)
+CFLAGS		= -g -O2 -Wall $(MACHDEP) $(INCLUDE)
+CXXFLAGS	= $(CFLAGS)
 
-LDFLAGS	=	-g $(MACHDEP) -Wl,-Map,$(notdir $@).map -T$(PWD)/ipl.ld
+LDFLAGS		= -g $(MACHDEP) -Wl,-Map,$(notdir $@).map -T$(PWD)/ipl.ld
```

Comment:
No actual changes here, please remove this from the diff.

Reply:
Yea whitespace only, but fixes the tab indentation so it aligns correctly (never was a tab fan).

Reply:
Ideally, yes. I'd say either do the whole file or just leave it alone. It's really no big deal, but I just don't want unrelated style changes to be in the same commit as it adds noise to the diff and makes it harder to read.

Reply:
Be careful btw, make really likes tabs for indentation, you can't always replace them with spaces.

Reply:
I suggest not messing with these as this come from devkitPro template makefiles.

Reply:
That's actually where these change came from. It seemed the Makefile had drifted from the template so I started over with the template and then applied the changes on top.

Reply:
Weird, because I did just that a few commits ago.

Reply:
Autoformatter/pretier/linter perhaps?

Reply:
Just leave it.

Reply:
+1 on keeping this in sync with dkPPC scripts then.